chord_metadata_service/patients/migrations/0005_remove_individual_comorbid_condition_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.5 on 2023-10-23 19:04
```

Comment:
these should get squashed into a v5_0_0 migration eventually for the v5 (phenov2) release